### DISCUSSION TITLE
Why mint fee per nft goes to object address?

### DISCUSSION THREAD

**Discussion by CryptainGolD** [aptos-labs/aptos-developer-discussions#474] [Questions] [answered]
I've published contract based on nft-minting-dapp template.
Once I create a collection with Mint fee per nft as 0.25 apt, and mint nfts, the mint fee goes to that collection address. How to turn it to collection creator's address?

**Comment by Tlazypanda** [comment, marked as answer]
gm gm @Cryptain-Harlock if you look into subdirectory `move/sources/launchpad.move` function `init_module` - the `mint_fee_collector_addr` is set to the signer address which you can modify to whatev you like

**Reply by CryptainGolD** [reply]
Thanks for that. I've managed it 🙏 🙏 🙏
